### DISCUSSION TITLE
OpenHASP without Home Assistant

### DISCUSSION THREAD

**Discussion by darkman7hr** [HASwitchPlate/openHASP#549] [Q&A] [answered]
Hi all,

I'm not using Home Assistant at all. I find it better to use node-red and mqtt.
Is there any way I can extract info to pages? Like date/time, wifi.. everything sent to status?

**Comment by smcgann99** [comment]
It's not really clear what you are asking, extract info from where ?

**Reply by darkman7hr** [reply]
To put it simply. Right now, I need date and time from the device itself, not being sent by home assistant or mqtt.
I can see device sending that info to mqtt but I cannot find way to include it in pages.jsonl

**Reply by smcgann99** [reply]
As far as I can see that is not possible, to update an element in the display you have to send the text to it via MQTT.

**Reply by darkman7hr** [reply]
So I need to send it, then send it back? :D damn :D I was hoping it's something simpler.

**Comment by Smart111222** [comment, marked as answer]
{ "page": 1, "id": 5, "obj": "label", "x": 240, "y": -13, "w": 240, "h": 100, "text_font": 100, "text_color": "#FFFF00", "align": "center", "text": "98:80:00", "template": "%H:%M" }
{ "page": 1, "id": 6, "obj": "label", "x": 260, "y": 100, "w": 240, "h": 32, "text_font": 28, "text_color": "#FFFF00", "align": "center", "text": "Wednesday", "template": "%d-%m-%Y" }
